### DISCUSSION TITLE
Some issues with the most recent codebase

### DISCUSSION THREAD

**Discussion by myrthos** [gethinode/hinode#508] [General]
There are two differences with the last Hinode codebase, which I just merged. And I don't think they are intended.

1) Frontmatter is shown on the list page. See the below screenshot.

![list](https://github.com/gethinode/hinode/assets/19652599/51fe16a2-63f6-4f0f-bb04-bba6e2cdf646)

This is the contents of my `_index.md file`:

```yaml
---
author: Joost Mans
title: Blog
nested: true
---
```

If I put some text as content, it will show that and not show the frontmatter anymore.

2) Using an icon instead of an image, has become a very large icon.

This is what it is now:

![project1](https://github.com/gethinode/hinode/assets/19652599/19153ece-8463-4fa0-a703-f24e0e2c7068)

This what it was:

![project2](https://github.com/gethinode/hinode/assets/19652599/a627ab53-95f4-4ec3-a005-e5b4a8653e04)

To be honest I don't know if these have happened in the updates of today or before, but it was not part of the last time I merged and published my site, which was Friday morning at 11:15.

**Comment by markdumay** [comment]
Upgrading to release v0.19.0 should fix it. Be sure to run npm i too. It’s related to Hugo issue #11406: https://github.com/gohugoio/hugo/issues/11406.

**Comment by markdumay** [comment]
Regarding the icon, I encountered a similar issue. I thought it was just me. ;-) add the following to your section configuration to fix it:

orientation = "horizontal-sm"

**Reply by markdumay** [reply]
See https://github.com/gethinode/hinode/blob/main/exampleSite/config/_default/params.toml

**Comment by myrthos** [comment]
`npm i` and modifying the orientation, indeed fixes the issues. Thanks
